azuredeploy.json

```diff
@@ -49,11 +54,11 @@
       "sku": {
         "name": "Basic",
         "tier": "Basic",
-        "capacity": 1
+        "capacity": 13
```

Comment:
Did you mean to change these properties of the 'Basic' SKU deployment?

Reply:
Basic SKU does not have auto-inflate enabled. Hence changed this to Standard SKU and made capacity 1 with auto-inflate enabled